src/components/AlbumSongsTable/AlbumSongTable.js

```diff
@@ -0,0 +1,111 @@
+import albumSongsTable from "./albumSongsTable.html";
```

Comment:
please check that folder name is the same as file name and matches class name

Reply:
Corrected